src/test/unit/variational/hier_logistic_test.cpp

```diff
@@ -89,7 +89,7 @@ class advi_test : public ::testing::Test {
 };
 
 TEST_F(advi_test, hier_logistic_constraint_meanfield) {
-  EXPECT_EQ(0, advi_->run(1, 2e4));
+  EXPECT_EQ(0, advi_->run("0.01", 1, 2e4, 50));
   SUCCEED() << "expecting it to compile and run without problems";
   EXPECT_NE("", output_stream_.str());
```

Comment:
beyond just this `EXPECT_NE`, are you expecting to see anything else? 

You might want to test the lower level function for other behavior and leave this as an `EXPECT_NE`, but I don't see any of those test around.